targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/trng_api_esp32.cpp

```diff
@@ -80,6 +85,10 @@ extern "C" int trng_get_bytes_esp32(uint8_t *output, size_t length, size_t *outp
     if (output_length != NULL) {
         *output_length = idx;
     }
+    if (idx == 0) {
+        memset(output, 0, length);
+    }
+    memset(recv_data, 0, sizeof(recv_data));
```

Comment:
This might be optimized away by the compiler, please consider using something similar to `mbedtls_zeroize`.